packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -150,6 +150,18 @@ export const render = async (req, res, next) => {
         })
     }
 
+    // Serialize the routes and add them to the config. We'll use this on the client-side later.
+    // TODO: we should not serialize the localized routes to reduce byte size
+    config.app.routes = routes.map((route) => {
```

Comment:
- consider only serializing route from `extendRoutes`
- move serialization logic to extension instead of base react-sdk
- force customers to create a new extension to use an async extendRoutes that includes warnings to make sure the customer uses fast functions in the impl. This will prevent customers from creating slow functions and reduce customer questions